src/Microsoft.TestPlatform.ObjectModel/Resources/Resources.resx

```diff
@@ -286,6 +286,10 @@
     <value>  {0}:
 {1}</value>
   </data>
+  <data name="TestResultTimerFormat" xml:space="preserve">
```

Comment:
This does have to be a resources.
Resources are strings that need to be localized. We should simply use a const for this.

Reply:
Understood and I agree. I'll make that update.

Reply:
Done in commit 1cbbb23d5b40eebcf18cecfdfcb89f32680ba93f